packages/phone_log/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.0.7
```

Comment:
When you change this; you also need to update pubspec.yaml.

Reply:
Done